src/routers/TabRouter.js

```diff
@@ -76,6 +76,7 @@ export default (
         state = {
           routes,
           index: initialRouteIndex,
+          isTransitioning: false,
```

Comment:
probably later we should integrate this properly in tab-view to handle animating tabs

Reply:
yes, absolutely!